CHANGELOG.md

```diff
@@ -49,6 +50,7 @@ IMPROVEMENTS:
 * api: Support VAULT_HTTP_PROXY environment variable to allow overriding the Vault client's HTTP proxy [[GH-12582](https://github.com/hashicorp/vault/pull/12582)]
 * auth/approle: The `role/:name/secret-id-accessor/lookup` endpoint now returns a 404 status code when the `secret_id_accessor` cannot be found [[GH-12788](https://github.com/hashicorp/vault/pull/12788)]
 * auth/approle: expose secret_id_accessor as WrappedAccessor when creating wrapped secret-id. [[GH-12425](https://github.com/hashicorp/vault/pull/12425)]
+* auth/kubernetes: `disable_iss_validation` and `issuer` configuration fields are deprecated. [[GH-127]](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/127)
```

Comment:
I think we'll want a couple entries for this, one under DEPRECATIONS and one under CHANGES, since both fields are being deprecated, and the default for `disable_iss_validation` changed.